README.md

```diff
@@ -421,6 +421,11 @@ Sends a message to the Game Coordinator requesting to create a lobby.  Listen fo
 * `[options]` - See paramter description in [#createPracticeLobby]
 * `[callback]` - optional callback, returns args: `err, response`.
 
+*Note: it is unknown what should be supplied to `tournament_game_id` (See
+[#414](https://github.com/Arcana/node-dota2/issues/414)), but ticketed lobbies can be created with
```

Comment:
When I tried it, creareTournamentLobby didn't work, even though I supplied the correct league id. I'll try again though when I can and report back.